content-src/experiments/send.yaml

```diff
@@ -28,7 +28,7 @@ measurements:
     Send is also subject to our <a>websites privacy notice</a>. When you visit the Send website, information such as your IP address is temporarily retained as part of a standard server log.
   - >
     Send will also collect information about the performance and your use of the service, such as how often you upload files, how long your files remain with Mozilla before they expire, any errors related to file transfers, and what cryptographic protocols your browser supports.
-
+measurements_l10nsuffix: '_v2_'
```

Comment:
Don't need underscores here, since it just gets camel-cased when it's turned into an ID. Should also probably be more like "filehash" to be more descriptive.